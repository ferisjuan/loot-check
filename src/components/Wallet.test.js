import React from 'react'
import { shallow } from 'enzyme'
import 'jest-enzyme'

import { Wallet } from './wallet'

describe('Wallet', () => {
	const props = { balance: 20 }
	const wallet = shallow(<Wallet {...props} />)

	it('renders', () => {
		expect(wallet).toMatchSnapshot()
	})

	it('displays the balance from props', () => {
		expect(wallet.find('.balance').text()).toEqual(
			`Wallet balance: ${props.balance}ðŸ˜Ž`
		)
	})
})
