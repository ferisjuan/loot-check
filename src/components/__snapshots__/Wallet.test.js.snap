// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wallet renders 1`] = `
<div>
  <h3
    className="balance"
  >
    Wallet balance: 
    20
    ðŸ˜Ž
  </h3>
</div>
`;
